src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -20,20 +18,18 @@ <h2 class="slds-text-heading_medium slds-text-heading--medium slds-hyphenate hea
                     </div>
                 </div>
             </template>
-            <template if:true={isFieldInfoReady}>
+            <template if:true={isRecordReady}>
```

Comment:
It seems the `isRecordReady` check is enough to check when the form fields should be rendered.